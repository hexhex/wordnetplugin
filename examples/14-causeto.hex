%% this returns all words that express,
%% what a word is cause to (give -> have)
%% POS: verbs

%% the search number for cause to
%% is 14.

word("give").
causeto(W,C) :- &wordnet[14,W](_,_,C,_), word(W).
