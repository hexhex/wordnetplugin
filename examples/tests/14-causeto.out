{causeto("give","eat"), causeto("give","hold"), causeto("give","have_got"), causeto("give","have"), word("give")}
